app/script/view_model/bindings/CommonBindings.js

```diff
@@ -126,73 +126,63 @@ ko.virtualElements.allowedBindings.stopBinding = true;
 /**
  * Resize textarea according to the containing text.
  */
-ko.bindingHandlers.resize = (function() {
-  let lastHeight = null;
-  let triggerValue = null;
-  let callback = null;
```

Comment:
The problem was that this IIFE is only called once during the lifecycle of the app.
Which means that this statefull variables (lastHeight, callback, triggerValue) are shared across all the instances of this binding.
Most of the time it works but whenever we create a new textarea with this binding, everything goes wrong